openpype/hosts/houdini/plugins/publish/collect_chunk_size.py

```diff
@@ -0,0 +1,40 @@
+import pyblish.api
+from openpype.lib import NumberDef
+from openpype.pipeline import OpenPypePyblishPluginMixin
+
+
+class CollectChunkSize(pyblish.api.InstancePlugin,
+                       OpenPypePyblishPluginMixin):
+    """Collect chunk size for cache submission to Deadline."""
+
+    order = pyblish.api.CollectorOrder + 0.05
+    families = ["ass", "pointcache",
+                "vdbcache", "mantraifd",
+                "redshiftproxy"]
+    hosts = ["houdini"]
+    targets = ["local", "remote"]
+    label = "Collect Chunk Size"
+    chunkSize = 999999
+
+    def process(self, instance):
+        attr_values = self.get_attr_values_from_data(instance.data)
+        instance.data["chunkSize"] = (
+            attr_values.get("chunkSize") or self.chunkSize
+        )
+
+    @classmethod
+    def apply_settings(cls, project_settings):
+        project_setting = project_settings["houdini"]["publish"]["CollectChunkSize"]         # noqa
+        cls.chunkSize = project_setting["chunk_size"]
+
+    @classmethod
+    def get_attribute_defs(cls):
+        return [
+            NumberDef("chunkSize",
+                      minimum=1,
+                      maximum=999999,
+                      decimals=0,
+                      default=cls.chunkSize,
+                      label="Frame Per Task")
+
+        ]
```

Comment:
Applying data is automated as long as you use the same key used in settings `"chunk_size"`

```suggestion
    label = "Collect Chunk Size"

    def process(self, instance):
        instance.data["chunkSize"] =  self.chunk_size

    @classmethod
    def get_attribute_defs(cls):
        return [
            NumberDef("chunkSize",
                      minimum=1,
                      maximum=999999,
                      decimals=0,
                      default=cls.chunk_size,
                      label="Frame Per Task")
        ]
```

Reply:
> Applying data is automated as long as you use the same key used in settings `"chunk_size"`

OP Studio settings apply there too. so the apply_setting function needs to be included. But I do agree with instance.data["chunkSize"] =  self.chunk_size